specification/cognitiveservices/data-plane/ImageSearch/v1.0/ImageSearch.json

```diff
@@ -0,0 +1,2171 @@
+            "type": "string",
+            "enum": [
+              "All",
+              "Small",
+              "Medium",
+              "Large",
+              "Wallpaper"
+            ],
+            "x-ms-enum": {
+              "name": "ImageSize",
+              "modelAsString": true
+            }
+          },
+          {
+            "name": "setLang",
+            "x-ms-client-name": "SetLang",
+            "in": "query",
+            "description": "The language to use for user interface strings. Specify the language using the ISO 639-1 2-letter language code. For example, the language code for English is EN. The default is EN (English). Although optional, you should always specify the language. Typically, you set setLang to the same language specified by mkt unless the user wants the user interface strings displayed in a different language. This parameter and the [Accept-Language](https://docs.microsoft.com/en-us/rest/api/cognitiveservices/bing-images-api-v7-reference#acceptlanguage) header are mutually exclusive; do not specify both. A user interface string is a string that's used as a label in a user interface. There are few user interface strings in the JSON response objects. Also, any links to Bing.com properties in the response objects apply the specified language.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "width",
+            "x-ms-client-name": "Width",
+            "in": "query",
+            "description": "Filter images that have the specified width, in pixels. You may use this filter with the size filter to return small images that have a width of 150 pixels.",
+            "required": false,
+            "type": "integer",
+            "format": "int32"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success.",
+            "schema": {
+              "$ref": "#/definitions/Images"
+            }
+          },
+          "default": {
+            "description": "An error has occurred. Check the response type and/or status code for more details.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful image search": {
+            "$ref": "./examples/SuccessfulImageSearchRequest.json"
+          }
+        }
+      }
+    },
+    "/images/details": {
+      "get": {
+        "summary": "The Image Detail Search API lets you search on Bing and get back insights about an image, such as webpages that include the image. This section provides technical details about the query parameters and headers that you use to request insights of images and the JSON response objects that contain them. For examples that show how to make requests, see [Searching the Web for Images](https://docs.microsoft.com/azure/cognitive-services/bing-image-search/search-the-web).",
+        "operationId": "Imagess_Details",
```

Comment:
Please remove the double "s" on Imagess.

Reply:
resolved